terminus-ui/drawer/src/drawer-footer.component.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  ViewEncapsulation,
+} from '@angular/core';
+
+/**
+ * The drawer footer UI Component
+ *
+ * @example
+ * <ts-drawer-footer>
+ *   THIS IS MY FOOTER
```

Comment:
Example docs indentation needs to be 13 spaces because of how TypeScript handles indents in docs.